### ISSUE TITLE
This extension breaks PHP snippets

### ISSUE THREAD

**Issue by mike2003** [DEVSENSE/phptools-docs#270]
1 After installation, I see my snippets and autocomplete tips twice in the list for PHP.
2 And also the popup with the preview stops showing. And if you call it manually, then it is not updated when you switch to another one.

**Comment by jakubmisek** [comment]
thank you for reporting the issue; 

1. VS Code is simply merging code completion from all the extensions together with the code snippets. I'm not sure we can behave better, but we'll try to find some solution.
2. Is it a popup with a preview of the snippets? May I aks for a screenshot - what is expected and what you see instead?

**Comment by mike2003** [comment]
1 Two identical snippets merging? Other formats don't have this problem.
2 This is normal behavior, and it goes away with this extension.
![](https://i.imgur.com/aceBQp8.jpg)

**Comment by jakubmisek** [comment]
I'm still not sure what's happening. It seems it works most of the time, but at some point, it stops ... 

Looking into it.

**Comment by jakubmisek** [comment]
ok, it has something to do with the following VSCode errors:

```
ERR Cannot destructure property 'detail' of 'H.completion.label' as it is null.: TypeError: Cannot destructure property 'detail' of 'H.completion.label' as it is null.
    at Object.getAriaLabel (vscode-file://vscode-app/c:/Program%20Files/Microsoft%20VS%20Code/resources/app/out/vs/workbench/workbench.desktop.main.js:745:114724)
    at K.renderElement (vscode-file://vscode-app/c:/Program%20Files/Microsoft%20VS%20Code/resources/app/out/vs/workbench/workbench.desktop.main.js:168:2366)
    at U.renderElement (vscode-file://vscode-app/c:/Program%20Files/Microsoft%20VS%20Code/resources/app/out/vs/workbench/workbench.desktop.main.js:168:2052)
    at p.W (vscode-file://vscode-app/c:/Program%20Files/Microsoft%20VS%20Code/resources/app/out/vs/workbench/workbench.desktop.main.js:135:15465)
    at p.Q (vscode-file://vscode-app/c:/Program%20Files/Microsoft%20VS%20Code/resources/app/out/vs/workbench/workbench.desktop.main.js:135:12396)
    at p.splice (vscode-file://vscode-app/c:/Program%20Files/Microsoft%20VS%20Code/resources/app/out/vs/workbench/workbench.desktop.main.js:135:10906)
    at vscode-file://vscode-app/c:/Program%20Files/Microsoft%20VS%20Code/resources/app/out/vs/workbench/workbench.desktop.main.js:3:218630
    at Array.forEach (<anonymous>)
    at t.splice (vscode-file://vscode-app/c:/Program%20Files/Microsoft%20VS%20Code/resources/app/out/vs/workbench/workbench.desktop.main.js:3:218617)
    at vscode-file://vscode-app/c:/Program%20Files/Microsoft%20VS%20Code/resources/app/out/vs/workbench/workbench.desktop.main.js:168:7633
    at m.bufferEvents (vscode-file://vscode-app/c:/Program%20Files/Microsoft%20VS%20Code/resources/app/out/vs/workbench/workbench.desktop.main.js:86:4470)
    at Q.splice (vscode-file://vscode-app/c:/Program%20Files/Microsoft%20VS%20Code/resources/app/out/vs/workbench/workbench.desktop.main.js:168:7609)
    at vscode-file://vscode-app/c:/Program%20Files/Microsoft%20VS%20Code/resources/app/out/vs/workbench/workbench.desktop.main.js:745:117448
workbench.desktop.main.js:620 
```

```
ERR Cannot read properties of undefined (reading 'find'): TypeError: Cannot read properties of undefined (reading 'find')
	at t.DefaultCompletionItemProvider.b (c:\Program Files\Microsoft VS Code\resources\app\extensions\emmet\dist\node\emmetNodeMain.js:1:195760)
	at t.DefaultCompletionItemProvider.provideCompletionItems (c:\Program Files\Microsoft VS Code\resources\app\extensions\emmet\dist\node\emmetNodeMain.js:1:194279)
	at _.provideCompletionItems (c:\Program Files\Microsoft VS Code\resources\app\out\vs\workbench\api\node\extensionHostProcess.js:100:51686)
	at c:\Program Files\Microsoft VS Code\resources\app\out\vs\workbench\api\node\extensionHostProcess.js:100:71536
	at ce.s (c:\Program Files\Microsoft VS Code\resources\app\out\vs\workbench\api\node\extensionHostProcess.js:100:63614)
	at ce.$provideCompletionItems (c:\Program Files\Microsoft VS Code\resources\app\out\vs\workbench\api\node\extensionHostProcess.js:100:71523)
	at c.N (c:\Program Files\Microsoft VS Code\resources\app\out\vs\workbench\api\node\extensionHostProcess.js:108:11655)
	at c.M (c:\Program Files\Microsoft VS Code\resources\app\out\vs\workbench\api\node\extensionHostProcess.js:108:11373)
	at c.H (c:\Program Files\Microsoft VS Code\resources\app\out\vs\workbench\api\node\extensionHostProcess.js:108:10415)
	at c.G (c:\Program Files\Microsoft VS Code\resources\app\out\vs\workbench\api\node\extensionHostProcess.js:108:9432)
	at c:\Program Files\Microsoft VS Code\resources\app\out\vs\workbench\api\node\extensionHostProcess.js:108:8220
	at E.invoke (c:\Program Files\Microsoft VS Code\resources\app\out\vs\workbench\api\node\extensionHostProcess.js:63:145)
	at h.deliver (c:\Program Files\Microsoft VS Code\resources\app\out\vs\workbench\api\node\extensionHostProcess.js:63:2121)
	at n.fire (c:\Program Files\Microsoft VS Code\resources\app\out\vs\workbench\api\node\extensionHostProcess.js:63:1729)
	at p.fire (c:\Program Files\Microsoft VS Code\resources\app\out\vs\workbench\api\node\extensionHostProcess.js:72:14916)
	at c:\Program Files\Microsoft VS Code\resources\app\out\vs\workbench\api\node\extensionHostProcess.js:124:16654
	at E.invoke (c:\Program Files\Microsoft VS Code\resources\app\out\vs\workbench\api\node\extensionHostProcess.js:63:145)
	at h.deliver (c:\Program Files\Microsoft VS Code\resources\app\out\vs\workbench\api\node\extensionHostProcess.js:63:2121)
	at n.fire (c:\Program Files\Microsoft VS Code\resources\app\out\vs\workbench\api\node\extensionHostProcess.js:63:1729)
	at p.fire (c:\Program Files\Microsoft VS Code\resources\app\out\vs\workbench\api\node\extensionHostProcess.js:72:14916)
	at MessagePortMain.<anonymous> (c:\Program Files\Microsoft VS Code\resources\app\out\vs\workbench\api\node\extensionHostProcess.js:124:14780)
	at MessagePortMain.emit (node:events:526:28)
	at MessagePortMain._internalPort.emit (node:electron/js2c/utility_init:5:364)
	at Object.callbackTrampoline (node:internal/async_hooks:130:17)
```

**Comment by jakubmisek** [comment]
I've fixed the bug above when snippets stopped working.

Looking into the problem with snippets shown twice (where the second one is without details)

**Comment by jakubmisek** [comment]
Fixed in the latest update `1.34`.

Thank you for your time reporting the issue, and please let us know if there would be anything else.
